.development.env

```
@@ -16,7 +16,7 @@ EQ_NEW_RELIC_ENABLED=False
 EQ_SECRETS_FILE=dev-secrets.yml
 EQ_KEYS_FILE=dev-keys.yml
 AWS_DEFAULT_REGION=eu-west-1
-GUNICORN_KEEP_ALIVE=2
+HTTP_KEEP_ALIVE=2
```

```suggestion
WEB_SERVER_KEEP_ALIVE=2
```

I don't think we should make an assumption about what protocol is being used for the keepalive

---

https://github.com/ONSdigital/eq-questionnaire-runner/pull/314#discussion_r496597536